employeeapplication/src/main/java/com/artimes/employee/controller/EmployeeController.java

```diff
@@ -0,0 +1,22 @@
+package com.artimes.employee.controller;
+
+import com.artimes.employee.model.Employee;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class EmployeeController {
+
+    @PostMapping("/employeeDetails")
+//    @ResponseBody
+    public ResponseEntity<String> employeeDetails(@RequestBody Employee employee){
+        System.out.println("Employee Id : "+employee.getEmpId());
```

Comment:
please add spacing before and afer +. Thats a good practice!!

Reply:
Thanks for the comment . Added space before and after +